test/WebSites/CustomRouteWebSite/Controllers/US/CustomRoute_ProductsController.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace CustomRouteWebSite.Controllers.US
+{
+    [Locale("en-US")]
+    public class CustomRoute_ProductsController : Controller
```

Comment:
The route sets `en-US` because the majority of people live in the US right? :trollface: We already have a pretty good sample of using catch-all in a route constraint.